src/graph/optimizer/Optimizer.cpp

```diff
@@ -30,12 +30,35 @@ StatusOr<const PlanNode *> Optimizer::findBestPlan(QueryContext *qctx) {
   auto optCtx = std::make_unique<OptContext>(qctx);
 
   auto root = qctx->plan()->root();
-  auto status = prepare(optCtx.get(), root);
-  NG_RETURN_IF_ERROR(status);
-  auto rootGroup = std::move(status).value();
+  auto spaceID = qctx->rctx()->session()->space().id;
+
+  // auto status = preprocess(root, qctx, spaceID);
+  // if (!status.ok()) {
+  //   LOG(ERROR) << "Failed to preprocess plan: " << status;
+  // }
+
+  auto ret = prepare(optCtx.get(), root);
+  NG_RETURN_IF_ERROR(ret);
+  auto rootGroup = std::move(ret).value();
 
   NG_RETURN_IF_ERROR(doExploration(optCtx.get(), rootGroup));
-  return rootGroup->getPlan();
+  auto *newRoot = rootGroup->getPlan();
+
+  auto status2 = postprocess(const_cast<PlanNode *>(newRoot), qctx, spaceID);
```

Comment:
Rename postprocess

Reply:
Why?

Reply:
postProcess

Reply:
Please see the dict, `postprocess` is a English word.
Also see https://rdrr.io/cran/TSPred/man/preprocess.html, https://docs.aws.amazon.com/sagemaker/latest/dg/model-monitor-pre-and-post-processing.html

Reply:
ACK.